scripts/dev/src2asciidoc.py

```diff
@@ -248,6 +248,8 @@ def _get_command_doc_notes(cmd):
         if cmd.no_cmd_split:
             yield ("* With this command, +;;+ is interpreted literally "
                    "instead of splitting off a second command.")
+        if cmd.no_replace_variables:
+            yield "* This command does not replace variables like +\{url\}+."
```

Comment:
Please use a raw string (`r"..."`) here or double the backslashes (I'm guessing you're using them because `{` and `}` have some special meaning in asciidoc?)